src/build-change-graph.js

```diff
@@ -91,8 +93,10 @@ async function buildChangeGraph({
     let tagCommit;
     if (fromCommit) {
       tagCommit = fromCommit;
+    } else if (sinceBranch) {
+      tagCommit = await getCommonAncestor('HEAD', sinceBranch, _package.cwd);
     } else {
-      tagCommit = await getCommitSinceLastRelease(_package);
+      tagCommit = await getCommitSinceLastRelease(_package, sinceBranch);
```

Comment:
Isn't `sinceBranch` `undefined` (or some other falsey value) here? Why pass it?

Reply:
Great catch! That was an earlier attempt.

Reply:
Fixed here https://github.com/CrowdStrike/monorepo-next/pull/137